python/lsst/ctrl/orca/DatabaseConfigurator.py

```diff
@@ -22,35 +22,35 @@
 #
 
 ##
-# @deprecated DatabaseConfigurator
+# DatabaseConfigurator
+#
+# Note: this is currently unused, but being kept in place for when
+# orchestration records provenance before runs
 #
 
 
 class DatabaseConfigurator(object):
+    """Configures a database for use.
 
-    ##
-    # @brief initialize
-    #
-    def __init__(self, runid, config):
+    @param runid - `str`
+        run identifier
+    @param databaseConfig - Config
+        database configuration object
+    @param prodConfig - Config
+        production configuration object
 
-        # run id
-        self.runid = runid
+    Note: this is currently unused, but being kept in place for when
+    orchestration records provenance for runs
+    """
```

Comment:
To be honest, I'm not sure if it is a must, but the Developer Guide seems to prefer [numpydoc](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt) format of docstrings.

```python
class DatabaseConfigurator(object):
    """Configures a database for use.

    Parameters
    ----------
    runid : `str`
        Run indetfier.
    ...

    Note
    ----
    ...
    """
```